web/client/translations/data.it-IT.json

```
@@ -1243,7 +1244,10 @@
             },
             "geometryError": "La configurazione della geometria non è valida",
             "annotationSaveGeometryError": "Il salvataggio è disabilitato a causa di una geometria non valida, correggilo per salvare l'annotazione",
-            "resolveAllErrors": "Si prega di correggere gli errori prima di modificare altre geometrie"
+            "resolveAllErrors": "Si prega di correggere gli errori prima di modificare altre geometrie",
+            "selectAnnotationFeature": "Seleziona una geometria annotation nel pannello di sinistra per editare coordinate e stili",
```

If we use "si prega" just before, let's use the same level of confidence  for the other strings.
```suggestion
            "selectAnnotationFeature": "Selezionare una geometria annotation nel pannello di sinistra per editare coordinate e stili",
```